install.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+ln -s $(pwd)/todo.sh ~/bin/todo
```

Comment:
```suggestion
ln -s $(pwd)/todo.sh ${INSTALL_DIR:-"~/bin/todo"}
```

I don't have `~/bin` but I have `~/.bin` in my `PATH`